modules/monitoring-searching-and-filtering-targets.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-targets.adoc
+
+[id="monitoring-searching-and-filtering-targets_{context}"]
+= Searching and filtering metrics targets
+
+The list of metrics targets can be long. You can filter and search these targets based on various criteria.
+
+== Understanding target filters
+
+In the *Administrator* perspective, the *Metrics Targets* page in the Targeting UI provides details about targets for default {product-title} and user-defined projects.
+
+This page lists information for each target, including the HTTP endpoint URL, the component being monitored, the status, the namespace, the last scrape time, and the duration of the last scrape.
+
+You can filter the list of targets by status and source. The following filtering options are available:
+
+* *Status* filters:
+** *Up*. The target is currently up and being actively scraped for metrics.
+** *Down*. The target is currently down and not being scraped for metrics.
+
+* *Source* filters:
+** *Platform*. Platform-level targets relate only to default {product-title} projects. These projects provide core {product-title} functionality.
+** *User*. User targets relate to user-defined projects. These projects are user-created and are customizable. 
+
+== Searching for targets
+
+In the *Administrator* perspective, the *Metrics Targets* page in the Targeting UI you can search for targets relating to default {product-title} and user-defined projects.
```

Comment:
I think we can remove this sentence since it is basically repeating the first sentence from the `Understanding target filters` section.

Maybe we could combine the `Understanding target filters` and `Searching for targets` sections, since they are all forms of filtering?

Reply:
Removed the sentence and combined with the previous section.